Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -403,7 +405,7 @@ public class DynamicHlsController : BaseJellyfinApiController
                 BreakOnNonKeyFrames = breakOnNonKeyFrames ?? false,
                 AudioSampleRate = audioSampleRate,
                 MaxAudioChannels = maxAudioChannels,
-                AudioBitRate = audioBitRate,
+                AudioBitRate = audioBitRate ?? Convert.ToInt32(Math.Min(maxStreamingBitrate ?? 128000, int.MaxValue)),
```

Comment:
```suggestion
                AudioBitRate = audioBitRate ?? maxStreamingBitrate ?? 128000,
```

Just make `maxStreamingBitrate` an `int`, no reason to have it as a `long`.
I would move the `128000` to a constant somewhere so we don't have (more) magic numbers

Reply:
Just let EncoderHelper to handle the default 128000.